src/AcquiaDrupalEnvironmentDetector.php

```diff
@@ -2,6 +2,8 @@
 
 namespace Acquia\DrupalEnvironmentDetector;
 
+use Symfony\Component\Console\Input\ArgvInput;
```

Comment:
You don't need this, given that you excluded some parts of the BLT method.

Reply:
👍